<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugSettingsWidget</class>
 <widget class="QWidget" name="DebugSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="import_logging">
            <property name="text">
             <string>Import logging</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="export_logging">
            <property name="text">
             <string>Export logging</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="shader_logging">
            <property name="text">
             <string>Shader logging</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="uniform_logging">
            <property name="text">
             <string>Uniform logging</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="save_color_surfaces">
            <property name="text">
             <string>Save color surfaces</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="texture_dumping">
            <property name="text">
             <string>Texture dumping</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="elf_dumping">
            <property name="text">
             <string>ELF dumping</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="watch_code">
            <property name="text">
             <string>Watch Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="watch_memory">
            <property name="text">
             <string>Watch Memory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="val_layers">
        <property name="text">
         <string>Validation Layer (Reboot required)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
